syntax = "proto3";
//import "google/protobuf/empty.proto";
import "common.proto";
import "component_proto/scene_comp.proto";
package regionservcie;
option cc_generic_services = true;

message StartCrossGsRequest
{
  IpPort rpc_client = 1;
  IpPort rpc_server = 2;
  uint32 gs_node_id = 3;
  uint32 server_type = 4;
}

message StartCrossGsResponse
{
  repeated SceneInfo scenes_info = 1;
}

message StartMSRequest
{
  IpPort rpc_client = 1;
  IpPort rpc_server = 2;
  uint32 ms_node_id = 3;
}

message StartMSResponse
{
}

service RgService 
{
  rpc StartCrossGs (StartCrossGsRequest) returns (StartCrossGsResponse);
  rpc StartMS (StartMSRequest) returns (StartMSResponse);
}
